package.json

```diff
@@ -93,5 +93,8 @@
     "terra-dev-site": "^2.0.0",
     "terra-toolkit": "^4.5.0",
     "webpack-merge": "^4.1.2"
+  },
+  "dependencies": {
+    "eslint-plugin-react": "7.11.0"
```

Comment:
Updated eslint-config-terra to latest version 2.1.0 since 2.0.0 was giving error on npm run lint:js. 
Added dependencies eslint-plugin-react : 7.11.0 explicitly since by default it was referring to 7.12.0 which was leading to below error : 
`AssertionError [ERR_ASSERTION]: Node must be provided when reporting error if location is not provided
    at assertValidNodeInfo (/Users/sm051274/Documents/GitHub/terra-core/node_modules/eslint/lib/util/report-translator.js:96:9)`

Reply:
I have a PR for this to lock down the eslint-plugin-react version in the eslint-config-terra module: https://github.com/cerner/eslint-config-terra/pull/19

Reply:
Just released estlint-config-terra v2.2.0 which has the change @emilyrohrbough mentioned. We can switch this to `^2.2.0`.

Reply:
Updated version with ^2.2.0 : https://github.com/cerner/terra-core/pull/2138/files#diff-32052204d6a35b2810d45b5391d555ed

Reply:
Is the version change still necessary for this PR?

Reply:
cc: @supreethmr

Reply:
it's not need. It was added to pass travis build previously.  
Reverted back to ^2.0.0 :  https://github.com/cerner/terra-core/pull/2138/commits/6ee41f00060f7fb20696ea4c157433122b468450